redwood.toml

```diff
@@ -1,7 +1,8 @@
 [web]
   port = 8910
   apiProxyPath = "/.netlify/functions"
-  includeEnvironmentVariables = ['AUTH0_DOMAIN', 'AUTH0_CLIENT_ID', 'AUTH0_REDIRECT_URI', 'AUTH0_AUDIENCE', 'AZURE_ACTIVE_DIRECTORY_CLIENT_ID', 'AZURE_ACTIVE_DIRECTORY_AUTHORITY', 'AZURE_ACTIVE_DIRECTORY_REDIRECT_URI', 'AZURE_ACTIVE_DIRECTORY_LOGOUT_REDIRECT_URI', 'MAGICLINK_PUBLIC', 'FIREBASE_API_KEY', 'FIREBASE_AUTH_DOMAIN', 'FIREBASE_DATABASE_URL', 'FIREBASE_PROJECT_ID', 'FIREBASE_STORAGE_BUCKET', 'FIREBASE_MESSAGING_SENDER_ID', 'FIREBASE_APP_ID', 'SUPABASE_KEY', 'SUPABASE_URL']
+  includeEnvironmentVariables = ['AUTH0_DOMAIN', 'AUTH0_CLIENT_ID', 'AUTH0_REDIRECT_URI', 'AUTH0_AUDIENCE', 'AZURE_ACTIVE_DIRECTORY_CLIENT_ID', 'AZURE_ACTIVE_DIRECTORY_AUTHORITY', 'AZURE_ACTIVE_DIRECTORY_REDIRECT_URI', 'AZURE_ACTIVE_DIRECTORY_LOGOUT_REDIRECT_URI', 'MAGICLINK_PUBLIC', 'FIREBASE_API_KEY', 'FIREBASE_AUTH_DOMAIN', 'FIREBASE_DATABASE_URL', 'FIREBASE_PROJECT_ID', 'FIREBASE_STORAGE_BUCKET', 'FIREBASE_MESSAGING_SENDER_ID', 'FIREBASE_APP_ID', 'SUPABASE_KEY', 'SUPABASE_URL', 'NHOST_BACKEND_URL']
```

Comment:
Ugh. Can we please make this multiline?

```
# Line breaks are OK when inside arrays
hosts = [
  "alpha",
  "omega"
]
```
https://github.com/toml-lang/toml/blob/master/README.md